api/src/main/resources/client_api.yaml

```diff
@@ -133,6 +138,33 @@ definitions:
           retrieve more results. If not set, there are no more results to retrieve.
         type: string
 
+  FieldSet:
+    type: object
+    description: >
+      A specification for fields to retrieve for a cohort. Exactly one of the properties below
+      should be specified.
+    properties:
+      tableQuery:
+        description: >
+          A query specifying how to pull data out of a single table.
+        $ref: "#/definitions/TableQuery"
+
+  TableQuery:
+    type: object
+    description: >
+      A specification for pulling data from a single table.
+    properties:
+      tableName:
+        description: >
+          The name of the table to retrieve data from.
```

Comment:
How would one figure out the set of possible values for this? Does this correspond to BigQuery table names? If so, what is the syntax? e.g. is it project.dataset.table format or whatever it is in the BigQuery query syntax?

Reply:
The project and dataset are determined by the CDR version you're working with.

The table name should match the BigQuery table... so for now, the tables in OMOP. We might at some point have the ability to query from other tables which aren't in OMOP (or even BigQuery). Either way we'll want some documentation of what tables, columns, etc. you can query from at some point.

In the meantime if you want me to be more specific here and say, give a link to the OMOP CDM docs, I can.

Reply:
Sure, a link to the table names in the CDM docs and a `, for example "persons"` or whatever a real example is, would be great.

Reply:
Done.